scripts/mkcloud

```diff
@@ -394,18 +394,19 @@ function onhost_create_cloud_lvm()
     fi
 
     echo "Checking for LVs treated by LVM as valid PV devices ..."
-    if lvmdiskscan | egrep "/dev/($cloud/|mapper/$cloud-)"; then
+    if lvmdiskscan | egrep "/dev/($cloudvg/|mapper/$cloudvg-)"; then
         error=$(cat <<EOF
 Error: your lvm.conf is not filtering out mkcloud LVs.
 Please fix by adding the following regular expressions
 to the filter value in the devices { } block within your
 /etc/lvm/lvm.conf file:
 
-    "r|/dev/mapper/$cloud.*|", "r|/dev/$cloud/.*|"
+    "r|/dev/mapper/$cloudvg-|", "r|/dev/$cloudvg/|", "r|/dev/disk/by-id/|"
```

Comment:
Why `/dev/disk/by-id/`?  Couldn't that filter out lots of devices which are unrelated to `mkcloud`?